engine/src/flutter/runtime/runtime_controller.h

```diff
@@ -636,6 +636,9 @@ class RuntimeController : public PlatformConfigurationClient,
   // |PlatformConfigurationClient|
   void UpdateSemantics(int64_t view_id, SemanticsUpdate* update) override;
 
+  // |PlatformConfigurationClient|
+  void SetApplicationLocale(const std::string& locale) override;
```

Comment:
Should we make this a `std::string` here and for all subsequent methods, and then `std::move` the string down?